app/models/book.py

```diff
@@ -16,15 +16,20 @@ def to_dict(self):
         book_as_dict["title"] = self.title
         book_as_dict["description"] = self.description
 
+        if self.author:
+            book_as_dict["author"] = self.author.name
+
```

Comment:
👍 This should really help folks with task list too. Though personally, I'm always on the fence about how much data from a related record to full into the "scalar" output of a different type.

Reply:
Agreed. I didn't want to grab the whole Author record, but in this case I figured someone using the API would benefit from having the name if they wanted to make further requests.

Reply:
In that case, I'd probably lean more towards including `author_id` and `author_name` as two field. But I think that's a little fussy for the example.